src/components/icons/IconPreview.svelte

```
@@ -28,8 +28,10 @@
 					class="flex max-h-[223px] w-full flex-grow justify-center p-5"
 					style="color: {customizableIcon.styles.glyphColor}; transform: scale({customizableIcon.styles
 						.iconScale}) translate({customizableIcon.styles.imgX}px, {customizableIcon.styles.imgY}px);">
-					{#if customizableIcon.svgContent}
-						{@html DOMPurify.sanitize(customizableIcon.svgContent)}
+					{#if customizableIcon.contentType === "image/svg+xml" && customizableIcon.svgContent}
+						{@html customizableIcon.svgContent}
```

ESLint

[error] `{@html}` can lead to XSS attack.

(svelte/no-at-html-tags)